### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by wangxy518**
updated a url

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ If your repo has certain guidelines for contribution, put them here ahead of the
 -->
 
 - [Contributor License Agreement](https://git.k8s.io/community/CLA.md) Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests
-- [Kubernetes Contributor Guide](http://git.k8s.io/community/contributors/guide) - Main contributor documentation, or you can just jump directly to the [contributing section](http://git.k8s.io/community/contributors/guide#contributing)
+- [Kubernetes Contributor Guide](https://github.com/kubernetes/community/tree/master/contributors/guide) - Main contributor documentation, or you can just jump directly to the [contributing section](http://git.k8s.io/community/contributors/guide#contributing)
```
Comment (justaugustus) [inline CONTRIBUTING.md]:
```suggestion
- [Kubernetes Contributor Guide](https://git.k8s.io/community/contributors/guide/README.md) - Main contributor documentation, or you can just jump directly to the [contributing section](https://git.k8s.io/community/contributors/guide/README.md#contributing)
```

Comment (wangxy518) [reply CONTRIBUTING.md]: No problem.

Comment (justaugustus) [reply CONTRIBUTING.md]: @wangxy518 -- did you have a chance to accept this suggestion? It would need to be committed for me to approve this PR.

**Comment by justaugustus** [comment]
Thanks @wangxy518!
/ok-to-test
/lgtm
/approve

**Comment by cjwagner** [comment]
/hold
https://prow.k8s.io/tide-history?repo=kubernetes%2Fenhancements&branch=master
> PR is unmergable. Do the Tide merge requirements match the GitHub settings for the repo? Squash merges are not allowed on this repository.

Please do not use the `tide/merge-method-squash` label in repos where squash merging is forbidden. Either remove the label or change the repo settings then `/hold cancel`.

**Comment by justaugustus** [comment]
@cjwagner -- thanks for the heads up. Didn't realize we didn't have squash on here.
/hold cancel

**Pull request merged**